dp/optimalStrategy.py

```diff
@@ -0,0 +1,53 @@
+# Python3 program to find out maximum

```

Comment:
Hey! This looks good. Can you please rename this file?

Reply:
Yes, sure! What do you want me to rename it to?

Reply:
If this is a specific question, then please move it in the codeforces column. Otherwise, name it as `coin_change_problem.py` ?